resources/src/macosX64Main/def/pluralizedString.def

```diff
@@ -0,0 +1,13 @@
+language = Objective-C
+package = dev.icerock.moko.resources.objc
```

Comment:
i think we should not duplicate it. 
create please sourceSet `appleMain` and fill it with this def and other apple sources.
and configure hierarhical source sets - macos and ios source sets should depends on appleMain. 
yes IDE will bad recognize imports, but it will be better than duplicating i think

Reply:
or we can do symlinks from all apple sourcesets to `appleMain` - we will not duplicate code, but IDE will correct recognize imports

Reply:
I've created the shared `appleMain` sourceSet - 1983e006fe7e9620b79613772519efec25199485
Unfortunately I still had to duplicate the `PluralFormattedStringDesc` & `PluralStringDesc` for macosX64 (the classes that use `pluralizedString.def`). Symlinking the whole target dir from iosX64 doesn't work, since the `FontResource` and `ImageResource` has different implementations and symlinking just the individual files resulted in:
```
moko-resources/resources/src/macosX64Main/kotlin/dev/icerock/moko/resources/desc/PluralStringDesc.kt'.
```

Reply:
I'm definitely open to suggestions for this issue

Reply:
i think we leave it as it is